packages/sdk/src/conversation/conversation.ts

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { BotFrameworkAdapter, TeamsInfo } from "botbuilder";
+import * as path from "path";
+import { NotificationTargetStorage, TeamsFxBotCommandHandler } from "./interface";
+import { CommandResponseMiddleware, NotificationMiddleware } from "./middleware";
+import { TeamsBotInstallation } from "./notification";
+import { ConversationReferenceStore, LocalFileStorage } from "./storage";
+
+/**
+ * Options to initialize {@link ConversationBot}.
+ *
+ * @beta
+ */
+export interface ConversationOptions {
+  /**
+   * A boolean, controlling whether to whether to include the notification feature.
+   * @defaultValue false
+   * (default: `false`).
+   */
+  enableNotification?: boolean;
+
+  /**
+   * An optional storage to persist bot notification connections.
+   *
+   * @remarks
+   * If `storage` is not provided, a default local file storage will be used,
+   * which stores notification connections into:
+   *   - ".notification.localstore.json" if running locally
+   *   - "${process.env.TEMP}/.notification.localstore.json" if `process.env.RUNNING_ON_AZURE` is set to "1"
+   *
+   * It's recommended to use your own shared storage for production environment.
+   *
+   * @beta
+   */
+  storage?: NotificationTargetStorage;
+
+  /**
+   * The command handlers to register with the underlying conversation bot that
+   * can process a command and return a response.
+   *
+   * @remarks
+   * If provided, the corresponding handler will be involked if the bot received a message
+   * that matches the command pattern (`string` or `RegExp`) defined in the handler.
+   */
+  commandHandlers?: TeamsFxBotCommandHandler[];
+}
+
+/**
+ * Provide static utilities for bot conversation, including
+ * - send notification to varies targets (e.g., member, channel, incoming wehbook)
+ * - handle command and response.
+ *
+ * @example
+ * Here's an example on how to send notification via Teams Bot.
+ * ```typescript
+ * // initialize (it's recommended to be called before handling any bot message)
+ * ConversationBot.initialize(adapter, {
```

Comment:
Could this class be designed as non-static one? Because we have removed global state in SDK and let developers initializing `TeamsFx` to store all configurations.

Reply:
Yes, there will be refactor of this class next week. Changes will be included in later PRs.
This PR is to expose conversation APIs to public npm package (alpha versions), to unblock the build and deploy stages.